content/applications/sales/sales/send_quotations/quote_template.rst

```diff
@@ -245,6 +245,58 @@ There is also a blue banner at the top of the quotation template design with a l
 return :guilabel:`Back to edit mode`. When clicked, Odoo returns to the quotation form in the
 back-end of the *Sales* application.
 
+Mass cancel quotations/sales orders
+===================================
+
+Odoo provides the ability to cancel multiple quotations (or sales orders) at once from the list view
+of the :guilabel:`Quotations` page (the main *Sales* app dashboard).
+
+To do so, first navigate to the :guilabel:`Quotations` page by going to :menuselection:`Sales app
+--> Orders --> Quotations`, or open the *Sales* app from the main Odoo dashboard.
+
+Once on the :guilabel:`Quotations` page, all the quotations and sales order are presented in a
+default list view.
+
+.. image:: quote_template/quotations-page.png
```

Comment:
I believe this image needs to be shrunk down to 760px or lower since it's currently 930px wide.

Reply:
So, ideally, you want to aim for something in the 760px range, but the true limit is 933px... so, taking that into consideration, I'm going to opt to keep these screenshots as-is, but if any other reviewer suggests the same, I'll be sure to implement the change (and give you 100% credit for the original suggestion 👍 )